src/core/Akka.MultiNodeTestRunner.Shared.Tests/TestRunShutdownSpec.cs

```diff
@@ -22,7 +23,7 @@ public class TestRunShutdownSpec : AkkaSpec
         [Fact]
         public void TestCoordinatorEnabledMessageSink_should_receive_TestRunTree_when_EndTestRun_is_received()
         {
-            var consoleMessageSink = Sys.ActorOf(Props.Create(() => new ConsoleMessageSinkActor(false)));
+            var consoleMessageSink = Sys.ActorOf(Props.Create(() => new ConsoleMessageSinkActor(true)));
```

Comment:
The spec said TestCoordinatorEnabled, but it is disabled in the original code (false), so I turn it on.